### ISSUE TITLE
Add specific style when highlight on ScrollViewListItem

### ISSUE THREAD

**Issue by Hova25** [onmotion/react-native-autocomplete-dropdown#109]
Hello,

Actually we can't customise style of ScrollViewListItem when text highlighting. I think we can add props for this
What do you think about this ? 
Do you want i create merge request with this update ?

King regards

**Comment by onmotion** [comment]
Hi @Hova25 you can implement your own element that will be used for rendering the ListItem https://github.com/onmotion/react-native-autocomplete-dropdown/blob/11101eeda08ffc15bcef9757b18eefc0c1276417/src/index.js#L224

**Comment by Hova25** [comment]
yes i now it but just for highlight styling we need to re-implement all functionalities of item rendering
